WalletWasabi.Fluent/ViewModels/Wallets/Send/PrivacyControlViewModel.cs

```diff
@@ -50,7 +51,9 @@ private async Task InitializeLabelsAsync()
 	{
 		var privateThreshold = _wallet.AnonScoreTarget;
 
-		await LabelSelection.ResetAsync(_wallet.Coins.GetPockets(privateThreshold).Select(x => new Pocket(x)).ToArray());
+		var cjManager = Services.HostedServices.Get<CoinJoinManager>();
```

Comment:
> I guess you don't want to propagate CoinJoinManager via the ctor because it leads to a cascade of changes.

Yes, exactly. This was the only way I found to make it work without rewriting ~50 tests and mocking out the CJM.

@soosr WDYT? Is there a better way to do this?

Reply:
> I believe that this goes against UI decoupling ideas.

It is true but doesn't really matter. Send hasn't been refactored yet and suffers several problems like this. I am ok with either the current or ctor solution too. @ichthus1604 will refactor it anyway.

Don't spend much time fixing this.